libuavcan/include/test/ptest_time.hpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Unit tests of time types and functions.
+ */
+/** @file */
+#ifndef UAVCAN_TEST_TIME_HPP_INCLUDED
+#define UAVCAN_TEST_TIME_HPP_INCLUDED
+
+#include "uavcan/uavcan.hpp"
+#include "uavcan/time.hpp"
+#include "gtest/gtest.h"
+
+namespace uavcan
```

Comment:
Oh, this reminds me. Suppose you're building an application which has a UAVCAN node in it (I mean, can you imagine?). You want the node to be stowed away into its very own namespace. How would you call it?

Say, **`uavcan`**? Oops, this one's already taken.

Should we not rename the library's namespace into `libuavcan`? Or would that be too long?

Reply:
okay. libuavcan